lab-taylor/gulpfile.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
This looks great, but your watch task is configured in an unusual manner.  It should be declared more like this:
gulp.task('watch', () => {
  gulp.watch(paths, [tasks])
});